docs/examples/config

```diff
@@ -11,6 +11,7 @@ sip_cafile		/etc/ssl/certs/ca-certificates.crt
 #sip_transports		udp,tcp,tls,ws,wss
 #sip_trans_def		udp
 sip_verify_server	yes
+#sip_verify_client	no
```

Comment:
is it an option to change this to "sip_verify_peer" that can be common for client/server ?

does the instance know if it is a client or server ?

Reply:
I think it would make sense to have both cases separately.

- `sip_verify_server`: If BareSIP connects to a peer, it checks the peer's (server) certificate and validates the hostname it is connecting to with the CN/SAN of the certificate presented.
- `sip_verify_client` If someone wants to connect to BareSIP, BareSIP sends a CertificateRequest to the peer and then verifies the peer's (client) certificate.

Client certificate verification is much less common, and most of the time only server verification is done. I don't think we should suddenly change this behavior and merge the two options. I could see users wanting one of the verifications done but not both.